File: Testes/ex1_slide_herancaMultipla/Material.cpp
#include "Material.hpp"

Material::Material(string nome, string autor) : titulo_material(nome), autor_material(autor){}

void Material::descricao()
{
    cout << "\nTitulo: " << this->titulo_material
        << "\nAutor: " << this->autor_material << endl;
}

File: Laboratórios/Java/ex1_slide_herancaMutlipla/src/ex1_slide_herancaMutlipla/Material.java
package ex1_slide_herancaMutlipla;

public class Material {
	protected String titulo_material;
	protected String autor_material;
	
	public Material(String titutlo, String autor) {
		this.titulo_material = titutlo;
		this.autor_material = autor;
	}

	public void descricao()
	{
		System.out.printf("\nTitulo: %s \nAutor: %s\n", this.titulo_material, this.autor_material);
	}
}
